client/src/pages/users/index.tsx

```diff
@@ -0,0 +1,269 @@
+
+The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+*/
+import { withPrefix } from "gatsby";
+import React, { useContext } from "react";
+import { useCookies } from "react-cookie";
+import { navigate } from "@reach/router";
+import {
+  AppBar,
+  CircularProgress,
+  IconButton,
+  MenuItem,
+  Paper,
+  Select,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableRow,
+  Toolbar,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import KeyboardArrowLeftIcon from "@material-ui/icons/KeyboardArrowLeft";
+import KeyboardArrowRightIcon from "@material-ui/icons/KeyboardArrowRight";
+import { fetchUsers, updateUserPermissions } from "api";
+import { Connection, Edge, User } from "types";
+import NavBar from "components/nav-bar";
+import { ColumnDef, ColumnHeader } from "components/column-header";
+import ToggleContext from "context/toggle";
+import withLocation from "wrap-with-location";
+import "styles/layout.css";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexFlow: "column",
+  },
+  container: {
+    flexGrow: 1,
+  },
+  appBar: {
+    height: "10%",
+    top: "auto",
+    bottom: 0,
+  },
+  progress: {
+    marginLeft: "50%",
+  },
+  paging: {
+    position: "absolute",
+    right: theme.spacing(1),
+  },
+}));
+
+const columns: ColumnDef[] = [
+  {
+    id: "name",
+    label: "Name",
+    minWidth: 170,
+    align: "left",
+    sortable: true,
+  },
+  {
+    id: "email",
+    label: "Email",
+    minWidth: 170,
+    align: "left",
+    sortable: true,
+  },
+  {
+    id: "role",
+    label: "Role",
+    minWidth: 170,
+    align: "center",
+    sortable: false,
+  },
+];
+
+const TableFooter = (props: {
+  classes: any;
+  hasNext: boolean;
+  hasPrev: boolean;
+  onNext: () => void;
+  onPrev: () => void;
+}) => {
+  const { classes, hasNext, hasPrev, onNext, onPrev } = props;
+
+  return (
+    <AppBar position="sticky" color="default" className={classes.appBar}>
+      <Toolbar>
+        <div className={classes.paging}>
+          <IconButton id="prev-page" disabled={!hasPrev} onClick={onPrev}>
+            <KeyboardArrowLeftIcon />
+          </IconButton>
+          <IconButton id="next-page" disabled={!hasNext} onClick={onNext}>
+            <KeyboardArrowRightIcon />
+          </IconButton>
+        </div>
+      </Toolbar>
+    </AppBar>
+  );
+};
+
+const UserItem = (props: {
```

Comment:
I think we have mostly settled on `function` for declaring React functional components? 

Whichever it is, is better if we're consistent across all opentutor react repos and definitely consistent at least within each individual project